tests/integration/base_object_test.go

```diff
@@ -51,20 +51,14 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	require.Nil(s.T(), cobsErr)
 	logger.Infof("Object store created successfully")
 
-	/* TODO: Reenable this test after we have the object user CRD
-
 	logger.Infof("Step 1 : Create Object Store User")
-	initialUsers, _ := helper.ObjectClient.ObjectListUser(storeName)
-	_, cosuErr := helper.ObjectClient.CreateUser(storeName, userid, userdisplayname)
+	cosuErr := helper.ObjectUserClient.Create(namespace, userid, userdisplayname, storeName)
 	require.Nil(s.T(), cosuErr)
-	usersAfterCreate, _ := helper.ObjectClient.ObjectListUser(storeName)
-	require.Equal(s.T(), len(initialUsers)+1, len(usersAfterCreate), "Make sure user list count is increased by 1")
-	getuserData, guErr := helper.ObjectClient.ObjectGetUser(storeName, userid)
-	require.Nil(s.T(), guErr)
-	require.Equal(s.T(), userid, getuserData.UserID, "Check user id returned")
-	require.Equal(s.T(), userdisplayname, *getuserData.DisplayName, "Check user name returned")
+	require.True(s.T(), helper.ObjectUserClient.UserSecretExists(namespace, userid, storeName), "make sure user secret was created")
 	logger.Infof("Object store user created successfully")
 
+	/* TODO: Reenable this test after we have the object user CRD
```

Comment:
it would be nice to use the generated user creds to do something against the s3 api, but that can be a separate PR unless you can think of something simple.

Reply:
Will change the comment in next commit. I'd love to get some s3 validation tests as well, but am not sure when I'd be able to get around to it right now.